lib/ansible/modules/cloud/vmware/vmware_guest_network.py

```diff
@@ -119,6 +119,8 @@
            When reconfigure MAC address, VM should be in powered off state.'
      - ' - C(connected) (bool): Indicates that virtual network adapter connects to the associated virtual machine.'
      - ' - C(start_connected) (bool): Indicates that virtual network adapter starts with associated virtual machine powers on.'
+     - ' - C(directpath_io) (bool): If set, Universal Pass-Through (UPT or DirectPath I/O) will be enabled on the network adapter.
+           UPT is only compatible for Vmxnet3 adapter. Clients can set this property enabled or disabled if ethernet virtual device is Vmxnet3.'
```

Comment:
`Clients can set this property enabled or disabled if ethernet virtual device is Vmxnet3.` seems redundant to me.